components/map/MapZoneBottomSheet.tsx

```diff
@@ -27,6 +28,9 @@ type Props = {
 const MapZoneBottomSheet = forwardRef<BottomSheet, Props>((props, ref) => {
   const { zone: selectedZone, setFlyToCenter, isZoomedOut, address } = props
 
+  const localRef = useRef<BottomSheet>(null)
+  const refSetter = useMultipleRefsSetter(localRef, ref)
```

Comment:
Please add a comment why this is needed. Otherwise we will not know in the future and it can be accidentally removed as not needed.